### ISSUE TITLE
Ensures links have discernible text

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2802]
**Search results page:**
Impact:
serious
Found on:
11/8/2021 at 8:18 PM
Found:
Automatically
Issue Tags:
cat.name-role-value
wcag2a
wcag412
wcag244
section508
section508.22.a
ACT
Issue Description
Ensures links have discernible text

Element location
.icon-rssfeed
Element source
<a class="icon-rssfeed" title="" data-toggle="tooltip" href="/catalog?format=atom&amp;q=&amp;search_field=all_fields&amp;sort=cataloged_tdt+desc%2C+pub_date_start_sort+desc%2C+title_sort+asc&amp;utf8=%E2%9C%93" data-original-title="Subscribe to results feed"> </a>
To solve this issue, you need to...

ACCEPTANCE CRITERIA:

- [ ] Fix at least one (1) of these issues:

- Element does not have text that is visible to screen readers

- aria-label attribute does not exist or is empty

- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

- Element has an empty title attribute

And

Fix the following:

- [ ] Element is in tab order and does not have accessible text
